std/http/file_server.ts

```diff
@@ -34,10 +34,7 @@ interface FileServerArgs {
 const encoder = new TextEncoder();
 
 const serverArgs = parse(args) as FileServerArgs;
```

Comment:
Nitpick: parsing of args should happen in `main()`.

Reply:
i've let it outsite of `main()` because it's used in `serveDir` https://github.com/denoland/deno/blob/64bd2768f7794408a9e8b7ba14edc990f3e2d0de/std/http/file_server.ts#L142

i don't know if i can change this method and send target as a function parameter to it, like this:
```js
async function serveDir(
  req: ServerRequest,
  dirPath: string,
  target: string
)
```